src/PostgREST/QueryBuilder.hs

```diff
@@ -418,7 +418,7 @@ pgFmtFilter table (Filter fld (Operation hasNot_ ex)) = notOp <> " " <> case ex
    (op, VText val) -> pgFmtFieldOp op <> " " <> case op of
      "like"  -> unknownLiteral (T.map star val)
      "ilike" -> unknownLiteral (T.map star val)
-     "@@"    -> "to_tsquery(" <> unknownLiteral val <> ") "
```

Comment:
is there any chance that 
`col @@ 'lala'::unknow` is the same as `col @@ to_tsquery('lala'::unknown)`?
i.e. postgres is doing the casting on it's own without the need for to_tsquery in which case that line should be removed? or is this autocasting a new feature of 9.5+ and not present in older versions?